### PR TITLE
Update paper.md

### PR DESCRIPTION
**Pull request by gonzex**
More removal of unneeded words.

### CODE REVIEW

File: JOSS/paper.md [hunk 1]
```diff
@@ -110,12 +110,12 @@ with $F^\text{even}(x)=F^\text{even}(-x)$ and $F^\text{odd}(x)=-F^\text{odd}(-x)
...
-[@Takatsuka2008;@kaltak2014low;@liu2016cubic]. For this minimax grid optimization, we use the Remez algorithm [@kaltak2014low] which is an iterative, numerically ill-conditioned procedure requiring high numerical precision. As the generation of the minimax parameters $\tau_j$, $\sigma_j$, $\omega_k$, $\gamma_k$ is tedious, the most feasible strategy is to tabulate the computed minimax parameters $\{\tau_j\}_{j=1}^n$, $\{\sigma_j\}_{j=1}^n$, $\{\omega_k\}_{k=1}^n$, $\{\gamma_k\}_{k=1}^n$ for their later use in LT-dMP2, RPA, and \textit{GW} calculations.
+[@Takatsuka2008;@kaltak2014low;@liu2016cubic]. This minimax grid optimization rely on Remez algorithm [@kaltak2014low], an iterative, numerically ill-conditioned procedure requiring high numerical precision. As the generation of the minimax parameters $\tau_j$, $\sigma_j$, $\omega_k$, $\gamma_k$ is tedious, the computed minimax parameters $\{\tau_j\}_{j=1}^n$, $\{\sigma_j\}_{j=1}^n$, $\{\omega_k\}_{k=1}^n$, $\{\gamma_k\}_{k=1}^n$ are tabulated for their later use in LT-dMP2, RPA, and \textit{GW} calculations.
```
Comment (dgolze) [inline JOSS/paper.md:116]: typo: should be "..relies on the Remez"

File: JOSS/paper.md [hunk 2]
```diff
@@ -77,23 +77,23 @@ While not being the main target of the library, the minimax time grids can also
...
+Low-scaling RPA and \textit{GW} algorithms include the Fourier transform of $\hat{\chi}^0(i\tau)$ to $\chi^0(i\omega)$ (blue dashed box in Fig. \ref{fig:flowchart}). The \textit{GW} space-time method performs two additional Fourier transforms: The screened Coulomb interaction $W(i\omega)$ is transformed to imaginary time (red dashed box), and the self-energy $\widehat{\Sigma}(i\tau)$ is Fourier transformed back to imaginary frequency (green dashed box).
 
-To convert between imaginary time and frequency grids, we introduce a nonuniform discrete cosine and sine transformation for even and odd functions $F^\text{even/odd}$, respectively [@liu2016cubic]. If the function $F$ is neither odd nor even we split the computation of functions $\hat{F}(i\tau)$ and $F(i\omega)$ into even and an odd parts [@liu2016cubic]
+Conversion between imaginary time and frequency grids rely on nonuniform discrete cosine and sine transformations for even and odd functions $F^\text{even/odd}$, respectively [@liu2016cubic]. If the function $F$ is neither odd nor even, computation of functions $\hat{F}(i\tau)$ and $F(i\omega)$ is split into even and an odd parts [@liu2016cubic]
```
Comment (dgolze) [inline JOSS/paper.md:90]: Typo: Either "The conversion ... relies  on" or "Conversions ... rely on"

**Review by dgolze — CHANGES_REQUESTED** [review]
found two typos (mismatch singular/pural forms)

**Review by Panadestein — APPROVED** [review]
This one also looks good now

**Review by dgolze — APPROVED** [review]
Looks good now

**Pull request merged**